<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>howreg</class>
 <widget class="QDialog" name="howreg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>163</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
background:#333;
border-radius:15px;
}



QLabel
{
font-family: century gothic;
font-size:24px;
color: white;
}

QPushButton
{
background: steelblue;
border-radius: 10px;
color: white;
}

QPushButton:pressed
{
background: teal;
border-radius: 10px;
color: white;
}</string>
  </property>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Участника</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Зарегистрироваться в качестве</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Эксперта</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: red;
border-radius: 10px;
}

QPushButton:pressed
{
background-color: darkred;
border-radius: 10px;
}

</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
